jedeschule/spiders/berlin.py

```diff
@@ -99,4 +100,24 @@ def parse_table(self, table):
         for row in table.css('tr'):
             for idx, value in enumerate(row.css('td')):
                 result[keys[idx]] = value.css('::text').extract_first()
-        return result
\ No newline at end of file
+        return result
+
+    # fix wrong tabs, spaces and new lines
+    def fix_data(self, string):
+        if string:
+            string = ' '.join(string.split())
+            string.replace('\n', '')
+            string.replace('\t', '')
+        return string
+
+    def normalize(self, item: Item) -> School:
+        return School(name=item.get('name'),
+                      id='BER-{}'.format(item.get('id')),
```

Comment:
This should be `BE` (derived from the ISO 3166-2:DE code) instead of `BER`.

Reply:
fixed to BE